enterprise/api/v1/serializers.py

```diff
@@ -214,3 +215,88 @@ def update(self, instance, validated_data):
     user = UserSerializer(read_only=True)
     enterprise_customer = EnterpriseCustomerSerializer(read_only=True)
     data_sharing_consent = UserDataSharingConsentAuditSerializer(many=True, read_only=True)
+
+
+class EnterpriseCatalogSerializer(serializers.Serializer):
+    """
+    Serializer for enterprise customer catalog.
+    """
+    # pylint: disable=invalid-name
+    id = serializers.IntegerField(read_only=True, help_text=_('Course catalog primary key.'))
+    name = serializers.CharField(help_text=_('Catalog name'))
+    query = serializers.CharField(help_text=_('Query to retrieve catalog contents'))
```

Comment:
This field should match with the course catalog field from course discovery, where it is `query`. Here is sample catalog from discovery service.
```
{
       "id": 12,
       "name": "Enterprise Dummy Catalog",
       "query": "*",
       "courses_count": 2258,
       "viewers": []
}
```